build/tag-stable.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+for theme in default bootstrap classic material nouvelle
```

Comment:
The list of themes could be moved to a separate file and `source`'d when needed. It is used in at least 3 locations in `/build` and can fall out of sync.

Reply:
Indeed. We just need to leave a comment in the file that **default** should always be first, so we can omit it when needed.

Reply:
done